### PR TITLE
Handle repos with inadequate information

### PR DESCRIPTION
**Pull request by adarsh**
* Inactivate repositories which lack privacy or org information
* Add rake task to update repo information

https://trello.com/c/oPFtmreZ/393-builds-are-being-run-for-active-repos-missing-privacy-and-organization-information

### CODE REVIEW

**Comment by adarsh** [comment]
@gylaz Note that I had to add the new `schema.rb` file. Please doublecheck before I merge. 707b26b

File: db/schema.rb [hunk 1]
```diff
@@ -11,10 +11,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20140926163029) do
+ActiveRecord::Schema.define(version: 20141219224840) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
+  enable_extension "pg_stat_statements"
```
Comment (gylaz) [inline db/schema.rb]: Did this sneak back in?

**Comment by gylaz** [comment]
Looks good.

File: db/schema.rb (hunk 1 above)
Comment (adarsh) [inline db/schema.rb]: Whoops again - yes, fixed in a094f80.

**Pull request merged**